### ISSUE TITLE
app.bndrun: resolve snapshots all the time from repo

### ISSUE THREAD

**Issue by maggu2810** [openhab/openhab-distro#928]
I wonder why the POM of the demo app still uses

```
    <repository>
      <id>openhab-artifactory-snapshot</id>
      <name>JFrog Artifactory Repository</name>
      <url>https://openhab.jfrog.io/openhab/libs-snapshot</url>
      <releases>
        <enabled>false</enabled>
      </releases>
      <snapshots>
        <enabled>true</enabled>
        <updatePolicy>always</updatePolicy>
      </snapshots>
    </repository>
```

Haven't you already changed `<updatePolicy>always</updatePolicy>` in the other places because you realized that slows down the dependency resolution all the time?

Or does it not have any impact in this case?
But why using it then?

**Comment by maggu2810** [comment]
Will be fixed by #929

**Comment by kaikreuzer** [comment]
This was indeed a left over, thanks for noticing.
